assets/styles/stack.css

```diff
@@ -89,7 +90,7 @@ body {
 
 /* Wrapper */
 .popup {
-    flex: 0 0 30rem
+    flex: 0 0 33rem
```

Comment:
I tried to be more respectful to the mockup made by Luc (and I thought it's looking better)